pyfakefs/fake_filesystem_unittest.py

```diff
@@ -152,6 +152,8 @@ def copyRealFile(self, real_file_path, fake_file_path=None,
                   offers the option to enable atime, and older versions of \
                   Linux may also modify atime.
         """
+        if not fake_file_path:
```

Comment:
Do you mean:
   `if fake_file_path is None`   ? 
`if fake_file_path is not None` would be the case where it already has a set value.